recipe/meta.yaml

```diff
@@ -19,6 +19,8 @@ requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
@conda-forge/core, this package already supports osx-arm64. Also, this PR only seems to repeat these lines, which I don't know if it's right.

So my question is: are the changes the bot is trying to merge here really necessary?

Reply:
No, you can just close the PR to mark the packages as migrated to `osx-arm64`.

Reply:
Also pinging @conda-forge/help-osx-arm64 about this (sorry, I just saw the handle in the bot message above).

Reply:
ok, thanks a lot @xhochy!